e2e/cypress/tests/support/notification.ts

```diff
@@ -2,9 +2,9 @@
 // See LICENSE.txt for license information.
 
 // Stub the browser notification API with the given name and permission
-export function spyNotificationAs(name, permission) {
-    cy.window().then((win) => {
-        function Notification(title, opts) {
+export function spyNotificationAs(name: string, permission: string) {
+    cy.window().then((win: any) => {
+        function Notification(title: string, opts?: NotificationOptions) {
```

Comment:
I'd recommend not to explicitly use `any` as much as possible. In this case, I'd let to use/infer the return type from `cy.window()`. With that, it could be like:
```javascript
export function spyNotificationAs(name: string, permission: NotificationPermission) {
    cy.window().then((win) => {
        win.Notification = Notification;
        win.Notification.requestPermission = () => Promise.resolve(permission);

        cy.stub(win, 'Notification').as(name);
    });

    cy.window().should('have.property', 'Notification');
}
```